modules/accredited_representative_portal/app/services/power_of_attorney_request_service/create/form_data_adapter.rb

```diff
@@ -0,0 +1,112 @@
+# frozen_string_literal: true
+
+module PowerOfAttorneyRequestService
```

Comment:
Nest all under `module AccreditedRepresentativePortal` here too. File will have to migrate too.